.github/workflows/cypress.yaml

```
@@ -0,0 +1,86 @@
+          exit 1
+
+  Run-Cypress-Tests:
+    timeout-minutes: 25
+    runs-on: ubuntu-latest
+    needs: Setup-Backend
+    env:
+      REACT_APP_TALAWA_URL: http://localhost:4000/graphql
+    steps:
+      - name: Checkout Frontend
+        uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)